packages/lib-webpack/src/webpack/PatchEntryCallbackPlugin.ts

```diff
@@ -0,0 +1,41 @@
+import { WebpackPluginInstance, Compiler, Compilation, sources, WebpackError } from 'webpack';
+import { findPluginChunks } from '../utils/plugin-chunks';
+
+export class PatchEntryCallbackPlugin implements WebpackPluginInstance {
+  constructor(
+    private readonly containerName: string,
+    private readonly callbackName: string,
+    private readonly pluginID: string,
+  ) {}
+
+  apply(compiler: Compiler) {
+    compiler.hooks.thisCompilation.tap(PatchEntryCallbackPlugin.name, (compilation) => {
+      compilation.hooks.processAssets.tap(
+        {
+          name: PatchEntryCallbackPlugin.name,
+          stage: Compilation.PROCESS_ASSETS_STAGE_ADDITIONS,
+        },
+        () => {
+          const { entryChunk } = findPluginChunks(this.containerName, compilation);
+
+          entryChunk.files.forEach((fileName) => {
+            compilation.updateAsset(fileName, (source) => {
+              const newSource = new sources.ReplaceSource(source);
+              const fromIndex = source.source().toString().indexOf(`${this.callbackName}(`);
+
+              if (fromIndex >= 0) {
+                newSource.insert(fromIndex + this.callbackName.length + 1, `'${this.pluginID}', `);
```

Comment:
@vojtechszocs could you explain what exactly we are doing here? I thought webpack would take care of the naming when using `callback` registration type, or the app would be responsible for this when using `custom`.

Reply:
Ah - nevermind. I see what's going on with the new diff above. Sorry for the confusion

Reply:
This is another file where I messed up the `git mv` sorry about that.